src/main/java/htsjdk/samtools/BAMFileReader.java

```diff
@@ -408,6 +408,21 @@ public BAMIndex getIndex() {
         return mIndex;
     }
 
+    /**
+      This method is added to support BAI in IGB [IGBF-1920]
```

Comment:
The method is added to support visualization of BAI index files in Integrated Genome Browser. See https://jira.transvar.org/browse/IGBF-1920.

Reply:
Also state where you copied this from originally.